test/livebook_teams/hubs_test.exs

```diff
@@ -2,61 +2,55 @@ defmodule Livebook.HubsTest do
   use Livebook.TeamsIntegrationCase, async: true
 
   alias Livebook.Hubs
-  @sample_hub "team-foo-bar-baz"
 
-  setup do
-    on_exit(fn -> Hubs.delete_hub(@sample_hub) end)
-    :ok
-  end
-
-  test "get_hubs/0 returns a list of persisted hubs" do
-    team = insert_hub(:team, id: @sample_hub)
+  test "get_hubs/0 returns a list of persisted hubs", %{user: user, node: node} do
+    team = create_team_hub(user, node)
     assert team in Hubs.get_hubs()
 
-    Hubs.delete_hub(@sample_hub)
+    Hubs.delete_hub(team.id)
     refute team in Hubs.get_hubs()
   end
 
-  test "get_metadata/0 returns a list of persisted hubs normalized" do
-    team = insert_hub(:team, id: @sample_hub)
+  test "get_metadatas/0 returns a list of persisted hubs normalized", %{user: user, node: node} do
+    team = create_team_hub(user, node)
     metadata = Hubs.Provider.to_metadata(team)
 
     assert metadata in Hubs.get_metadatas()
 
-    Hubs.delete_hub(@sample_hub)
+    Hubs.delete_hub(team.id)
     refute metadata in Hubs.get_metadatas()
   end
```

Comment:
This would fail because we were creating a hub only in Livebook, but not in the hub server, so as soon as it tried to connect `TeamClient` would fail and remove the hub.